tests/unit/components/playback/controls/EditCustomer.spec.js

```diff
@@ -0,0 +1,514 @@
+      const selector_id_suffix_nickname_id = "nickname-id";
+
+      const editcustomer = {
+        uuid: "",
+        apikey: "",
+        nickname: "",
+      };
+
+      editcustomer.uuid = uuid;
+      editcustomer.apikey = apikey;
+      editcustomer.nickname = nickname;
+
+      const propsData = {
+        dialogdata: editcustomer,
+        dataindex: 0,
+      };
+      wrapper = mount(ComponentToTest, {
+        propsData,
+        store,
+        localVue,
+      });
+
+      const target_input_field_uuid = wrapper.find(
+        "#input-widget-field-" + selector_id_suffix_alphanumeric_id
+      );
+      const target_error_message_uuid = wrapper.find(
+        "#input-widget-feedback-" + selector_id_suffix_alphanumeric_id
+      );
+      target_input_field_uuid.setValue(uuid);
+      await Vue.nextTick();
+
+      expect(target_error_message_uuid.text()).toStrictEqual(invalid_uuid);
+
+      const target_input_field_apikey = wrapper.find(
+        "#input-widget-field-" + selector_id_suffix_apikey_id
+      );
+      const target_error_message_apikey = wrapper.find(
+        "#input-widget-feedback-" + selector_id_suffix_apikey_id
+      );
+      target_input_field_apikey.setValue(apikey);
+      await Vue.nextTick();
+
+      expect(target_error_message_apikey.text()).toStrictEqual(invalid_apikey);
+
+      const target_input_field_nickname = wrapper.find(
+        "#input-widget-field-" + selector_id_suffix_nickname_id
+      );
+      const target_error_message_nickname = wrapper.find(
+        "#input-widget-feedback-" + selector_id_suffix_nickname_id
+      );
+      target_input_field_nickname.setValue(nickname);
+      await Vue.nextTick();
+
+      expect(target_error_message_nickname.text()).toStrictEqual(
+        invalid_nickname
+      );
+
+      const target_button_label_btn = wrapper.findAll(".span__button_label");
+      const cancel_btn = target_button_label_btn.at(0);
+      expect(cancel_btn.attributes().style).toContain(
+        "color: rgb(255, 255, 255);"
+      );
+      const delete_btn = target_button_label_btn.at(1);
+      expect(delete_btn.attributes().style).toContain(
+        "color: rgb(255, 255, 255);"
+      );
+      const save_btn = target_button_label_btn.at(2);
+      expect(save_btn.attributes().style).toContain(save_btn_css);
+    }
+  );
+});
+
+describe("EditCustomer.clicked_button", () => {
+  beforeEach(async () => {
+    store = await NuxtStore.createStore();
+  });
+
+  beforeAll(async () => {
+    // note the store will mutate across tests, so make sure to re-create it in beforeEach
+    const storePath = `${process.env.buildDir}/store.js`;
+    NuxtStore = await import(storePath);
+  });
+
+  afterEach(() => wrapper.destroy());
+
+  test.each([
+    [
+      "5FY8KwTsQaUJ2KzHJGetfE",
+      "06ad547f-fe02-477b-9473-f7977e4d5e17",
+      "Experiment anemia -1",
+      "",
+      "",
+      "",
+      "color: rgb(255, 255, 255); width: 166.66666666666666px; left: 333.3333333333333px;",
```

Comment:
If you are checking CSS, use the Jest `.contains` to only test for the CSS property you are interested in---not the properties that are constant or unrelated to this unit you are testing

Reply:
Corrected `toContains` and verification now only for property of color.